server/api.go

```diff
@@ -79,7 +79,22 @@ func (p *Plugin) PublishStatusChanged(w http.ResponseWriter, r *http.Request) {
 	}
 
 	statusChangedEvent.Email = user.Email
-	p.wsPool.Broadcast <- statusChangedEvent
+	p.BroadcastEvent(statusChangedEvent)
+
+	eventBytes, marshalErr := json.Marshal(statusChangedEvent)
+	if marshalErr != nil {
+		p.API.LogDebug("Error in marshaling the status changed event", "Error", marshalErr.Error())
```

Comment:
```suggestion
		p.API.LogDebug("Error in marshaling the status for changed event", "Error", marshalErr.Error())
```

Reply:
Your suggestion is wrong grammatically and it is changing the meaning of the line.

Reply:
okay so `changed event` is the name of a status?

Reply:
`status changed` is the name of the event.

Reply:
should not we put the text `status changed` as into quotes `" "` as it's an event name and other can also misunderstand this